resources/instructions/createSObjectLwcQuickActions.html

```diff
@@ -1,81 +1,110 @@
 <!doctype html>
 <html lang="en">
-    <head>
```

Comment:
Sorry, most of the changes is just the addition of the scripts in the html. I made mistake of reformatting doc without realizing.